### PR TITLE
Publishing Specification meeting minutes for October.

### PR DESCRIPTION
**Pull request by TanjaObradovic**
Please merge this as soon as you can. Thanks!

### CODE REVIEW

File: data/meeting_minutes.yml [hunk 1]
```diff
@@ -347,6 +347,31 @@ items:
...
+          url: specification_committee/minutes-specification-october-28-2020.pdf
+          title: October 28, 2020 (pdf)
+          year: 2020
+       -
+          url: specification_committee/minutes-specification-october-21-2020.pdf
+          title: October 21, 2020 (pdf)
+          year: 2020
+       -
+          url: specification_committee/minutes-specification-october-14-2020.pdf
+          title: October 17, 2020 (pdf)
```
Comment (ericpoirier) [inline data/meeting_minutes.yml:364]: Here it should say October 14, 2020

File: data/meeting_minutes.yml [hunk 2]
```diff
@@ -347,6 +347,31 @@ items:
...
+          year: 2020
+       -
+          url: specification_committee/minutes-specification-october-7-2020.pdf
```
Comment (ericpoirier) [inline data/meeting_minutes.yml:367]: You have a typo in the actual name of your file. There's an extra space
"minutes-specification-october- 7-2020.pdf"

You will have to update the name of your file.

**Review by ericpoirier — COMMENTED** [review]
Hi Tanja,

Here are a few comments that will need to be looked at before we can merge your PR.

Thanks.

File: data/meeting_minutes.yml (hunk 1 above)
Comment (chrisguindon) [reply data/meeting_minutes.yml]: I made the fix

**Pull request merged**